bundle/DependencyInjection/Compiler/ScheduledVisibilityHandlerRegistrationPass.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Bundle\IbexaScheduledVisibilityBundle\DependencyInjection\Compiler;
+
+use LogicException;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Definition;
+use Symfony\Component\DependencyInjection\Reference;
+
+use function sprintf;
+
+final class ScheduledVisibilityHandlerRegistrationPass implements CompilerPassInterface
+{
+    private string $handlerRegistryId = 'Netgen\Bundle\IbexaScheduledVisibilityBundle\ScheduledVisibility\Registry';
+    private string $handlerTag = 'netgen.ibexa_scheduled_visibility.handler';
+
+    public function process(ContainerBuilder $container): void
+    {
+        if (!$container->has($this->handlerRegistryId)) {
+            return;
+        }
+
+        $handlerRegistryDefinition = $container->getDefinition($this->handlerRegistryId);
+
+        $handlers = $container->findTaggedServiceIds($this->handlerTag);
+
+        foreach ($handlers as $id => $attributes) {
+            $this->registerResolver($handlerRegistryDefinition, $id, $attributes);
+        }
+    }
+
+    private function registerResolver(Definition $handlerRegistryDefinition, string $id, array $attributes): void
```

Comment:
Rename to `registerHandler()`.

Reply:
fixed here 5f99ea81463f2af3c3a7ba5d161c93cf578c1676